cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -214,7 +214,12 @@ install_cross_toolchain_pkg() {
   info "Configuring environment variables for cross-compilation"
   export PATH="/build/bin:${PATH}"
   export SYSROOT="/build/usr/x86_64-cros-linux-gnu"
-  export CC="x86_64-cros-linux-gnu-gcc"
+  local kernel_version="$(uname -r | grep -o "^[0-9]*\.[0-9]*" | head -1)"
+  if [[ "${kernel_version}" = 4.14 ]]; then
```

Comment:
Please also include 4.4. Although all 4.4 images are deprecated, we want the cos-gpu-installer to be backward compatible. 

Also add a comment to indicate COS-kernel will be compiled by clang starting from 4.19.

Reply:
I feel like this should not be kernel version based, but COS image version based. For example, what if we roll-back CLANG for 4.19? Or rollout a 4.14 based image with CLANG?
Also, the indication whether which compiler is used for a particular image version should come from either the image itself (ex: `/etc/toolchain`) or from the corresponding artifacts in `gs://cos-tools/<build#>`.

Reply:
I agree. We should not determine the toolchain used to build the kernel from the kernel version and/or name.